### PR TITLE
Update Getting Started 

### PR DESCRIPTION
**Pull request by MekWarrior**
This PR should allow for us to begin final documentation tests and ask other labs to assist in robustness testing. As prep for this, I have updated the wrapper script command to include a new version (move from 1.0.0 to 1.1.0). So we should release a new version before this PR is approved.

Additionally, this PR: 
Drops the numbered format from the original readme and included it only where it adds to readability. Eliminated outdated Google console information. Copy-edited for grammar, consistency, and clarity.

### CODE REVIEW

**Comment by dylanbannon** [comment]
Just a versioning note: If we merge this into `stable`, then it'll be queued up as part of the next `stable` to `master` merge. Since every merge from `stable` into `master` is supposed to mean a new version for the project, this'll be part of the next Kiosk version. If we designate the next release as `1.1.0`, then everything works out exactly like @MekWarrior requested.

File: docs/source/GETTING_STARTED.rst [hunk 1]
```diff
@@ -112,17 +110,21 @@ You are now ready to start the kiosk! Those interested in Kiosk development shou
...
 
-11. Find the cluster's web address by choosing the ``View`` option form the Kiosk's main menu. (Depending on your chosen cloud provider and the cloud provider's settings, your cluster's address might be either a raw IP address, e.g., "123.456.789.012", or a URL, e.g., "deepcellkiosk.cloudprovider.com".)
+* Find the cluster's web address by choosing the ``View`` option form the Kiosk's main menu. (Depending on your chosen cloud provider and the cloud provider's settings, your cluster's address might be either a raw IP address, e.g., "123.456.789.012", or a URL, e.g., "deepcellkiosk.cloudprovider.com".)
 
-12. Go to the cluster address in your web browser to find the DeepCell Kiosk frontpage. To run a job (load raw data and download the results) use the ``PREDICT`` tab.
+* Go to the cluster address in your web browser to find the DeepCell Kiosk frontpage. To run a job (load raw data and download the results) use the ``PREDICT`` tab.
```
Comment (willgraf) [inline docs/source/GETTING_STARTED.rst]: I think this `PREDICT` should look more like `Predict`.

**Comment by willgraf** [comment]
Finally, one more comment:

How would this section look:
> Find the cluster's web address by choosing the View option form the Kiosk's main menu. (Depending on your chosen cloud provider and the cloud provider's settings, your cluster's address might be either a raw IP address, e.g., "123.456.789.012", or a URL, e.g., "deepcellkiosk.cloudprovider.com".)

Maybe the quotes could be be formatted instead of just strings.

**Comment by willgraf** [comment]
Finally, the license badge is pointed to the official Apache 2 license, but our software uses a modified apache 2 license.

File: docs/source/GETTING_STARTED.rst (hunk 1 above)
Comment (MekWarrior) [reply docs/source/GETTING_STARTED.rst]: Done

**Comment by MekWarrior** [comment]
> This looks great to me, nice content updates.
> 
> While we're here in this PR, can we update the screenshots to have the same size? they are different sizes but right next to eachother and I find it very jarring.

This is a markdown vs rst issue. It looks ok in RTD but not on github. The images are now the same size but the issue remains.

**Pull request merged**